Lib/ssl.py

```diff
@@ -781,17 +781,16 @@ def __init__(self, sock=None, keyfile=None, certfile=None,
         self.do_handshake_on_connect = do_handshake_on_connect
         self.suppress_ragged_eofs = suppress_ragged_eofs
         if sock is not None:
-            socket.__init__(self,
-                            family=sock.family,
+            super().__init__(family=sock.family,
```

Comment:
Please fix indention

Reply:
Should be done. One attribute per line seems to be the style.